src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -784,7 +791,7 @@ class Polyhedron
     // Expanded swap operations for HIP workaround.
     if(checkSign)
     {
-      double vol = poly.volume();
+      double vol = poly.signedVolume();
       if(vol < 0)
```

Comment:
Minor: since `vol` is not used elsewhere, it might be better to fold this into a single line
```suggestion
      if(poly.signedVolume() < 0)
```